cves/kernel/CVE-2020-10781.yml

```diff
@@ -235,11 +231,11 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
+  - commit: 
```

Comment:
Where there any interesting commits before the fix? For my CVEs both had a few commits that moved around the vulnerability. Just curious if you found anything interesting.

Reply:
All that I could find were the vcc commit and the fix commit for this one. Honestly it was a very bland bug with not a lot going on! That's ok.